opensource.md

```diff
@@ -0,0 +1,36 @@
+<!-- We want a table of contents in the sidebar.  The krampdown -->
+<!-- renderer supports {:toc}, but there's no way to have a layout -->
+<!-- processed by krampdown.  Thus, we put some light layout -->
+<!-- information here in the markdown. -->
+{% include sidebar.html %}
+
+<section markdown="1">
+This is the `How to Open Source like a Trussel`, written by Trussels for Trussels. We share it here for others to see how we approach open source.
+
+# Default To Open
+Truss encourages working in the open. What does this mean? Making the software we write open source where we can as often as we can so that everyone can benefit. Benefits include early identification and elimination of defects, cost savings through outside involvement, and reusability.
+
+## Exceptions
+As with everything, there are always going to be certain exceptions that just don't fit the model of working in the open. This should be rare, but take this into consideration before opening up your repo to the general public.
+
+- Data published contains sensitive information.
+- We do not have the rights to reproduce or publish the item on our own.
+- Publishing is restricted by law or other regulations, such as the Export Administration Regulations.
+
+<!--
+## Oh no, I've published an excepted case, what do?
+TODO
+-->
+
+# Community First
+One of the things that is imortant to us is making sure that we have a community around our projects that is safe for everyone to participate, no matter if they are fellow Trussels or not. This means that we have developed a strong [Code of Conduct](CODE-OF-CONDUCT) that governs how we manage the community and contributions to our projects. Our ultimate goal is to make sure that everyone has a voice and is able to contribute to our software in ways that are meaningful to them. You should always include a copy of the Code Of Conduct, or point to this canonical version so that the community at large is aware.
+
+# Licensing
+
+Documentation should be licensed as [CC-BY 4.0 International](LICENSE-CC-BY-4.0) when possible.
+
+Where it makes sense, source code should default to using a [BSD](LICENSE-BSD-3-CLAUSE) license.
```

Comment:
This isn't correct. We have a TDR about licensing that should be followed for Truss related open source work: https://docs.google.com/document/d/12UYIN3XfRPdKQV87_0ILa9-z6eWRBcLqtwP9fCyR6Tg/edit

Reply:
Not sure if you want to summarize, or link to the TDR from here.

Reply:
Can do both. I hadn't seen that explicitly, and was going mostly off most of the terraform modules that have license files included that have been published. Assumptions get you ever time.

Reply:
Heh. Yeah, I figured as much. For terraform, those should be licensed to fit that ecosystem which probably isn't Apache 2.0. Same with our React project(s). Turns out licensing is subtle and complex! :)

Reply:
I _think_ I've addressed this properly now.